packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -82,8 +75,17 @@ export const isIcon = ( {
 	};
 
 	return css`
+		color: ${ COLORS.gray[ 900 ] };
 		width: ${ iconButtonSizes[ size ] };
 		padding-left: 0;
 		padding-right: 0;
 	`;
 };
+
+export const buttonActive = css`
```

Comment:
Moved `buttonActive` styles after the `isIcon` styles, so that the active color will still take precedence (thus making the icon's color white when it's selected/active)